03-bash/task1.sh

```diff
@@ -0,0 +1,204 @@
+#!/bin/bash
+# Version 1.0.0
+
+# print help information on to a screen
+help_usage(){
+	echo "Usage ./task1.sh [-h|--help] [-n num] [file] "
+	echo "-h, --help - number of resulting file"
+	echo "-n         - number of files with hardware info"
+	echo "file       - file name with obtained hardware info"
+	echo "For example, ./task1.sh -n 9 ./test/info/info.txt"
+}
+
+# check and create work directory
+chk_work_dir(){
+	if ! [[ -d "$1" ]]; then
+	mkdir -p $1
+		if ! [ -d "$1" ]; then
+			>&2 echo "Error, working directory $1 was not created. Exit  with error."
+			exit 1
+		fi
+	else
+		echo "Directoy $1 is exists."
```

Comment:
Typing error 'Directory'. Syntax error 'is exists'. Should be 'exits' (verb in Present Simple)